README.md

```diff
@@ -269,11 +269,12 @@ Detailed information regarding Cygnus can be found in the [Installation and Admi
   <tr><td>Time and size-based data management policies</td><td>0.13.0</td></tr>
   <tr><td rowspan="2">OrionMySQLSink</td><td>First implementation</td><td>0.2.0</td></tr>
   <tr><td>Batching mechanism</td><td>0.10.0</td></tr>
-  <tr><td rowspan="5">OrionSTHSink</td><td>First implementation</td><td>0.8.0</td></tr>
+  <tr><td rowspan="6">OrionSTHSink</td><td>First implementation</td><td>0.8.0</td></tr>
   <tr><td>Hash based collections</td><td>0.8.1</td></tr>
   <tr><td>TimeInstant metadata as reception time</td><td>0.12.0</td></tr>
   <tr><td>Batching mechanism</td><td>0.13.0</td></tr>
   <tr><td>Time and size-based data management policies</td><td>0.13.0</td></tr>
+  <tr><td>String-based aggregation (ocurrences)</td><td>0.14.0</td></tr>
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/2d262bc5171759041c2b93dda3cc088c299d32d2